extensions/prototypes/labgraph_monitor/src/components/SettingPanel/EdgeSettings.tsx

```diff
@@ -62,20 +93,64 @@ const Edge: React.FC = (): JSX.Element => {
-                                                        {type}
+                                                        {field[1].type}
                                                     </TableCell>
                                                 </TableRow>
                                             );
                                         }
                                     )}
+                                    <Button onClick={handleToggle}>
+                                        {open ? 'Show less' : 'Show more'}
+                                    </Button>
+
+                                    {Object.entries(message['fields']).map(
+                                        (field, index) => {
+                                            return (
+                                                <TableRow key={index}>
+                                                    {open && (
+                                                        <TableCell>
+                                                            {field[0]}
+                                                        </TableCell>
+                                                    )}
+
+                                                    {!open ? null : connection ===
+                                                      WS_STATE.CONNECTED ? (
+                                                        <TableCell
+                                                            style={{
+                                                                whiteSpace:
+                                                                    'normal',
+                                                                wordBreak:
+                                                                    'break-word',
+                                                            }}
+                                                        >
+                                                            {field[1].content +
+                                                                ', '}
+                                                        </TableCell>
+                                                    ) : (
+                                                        <TableCell
+                                                            style={{
+                                                                whiteSpace:
+                                                                    'normal',
+                                                                wordBreak:
+                                                                    'break-word',
+                                                            }}
+                                                        >
+                                                            {mockData.join(' ')}
+                                                        </TableCell>
+                                                    )}
```

Comment:
This nested ternary is a bit hard to follow
```suggestion
                                                    {open ? (
                                                        <TableCell
                                                            style={{
                                                                whiteSpace:
                                                                    'normal',
                                                                wordBreak:
                                                                    'break-word',
                                                            }}
                                                        >
                                                            {connection === WS_STATE.CONNECTED ? 
                                                                 `${field[1].content}, ` : mockData.join(' ')
                                                             }
                                                        </TableCell>
                                                    ) : null}
```